content/docs/command-reference/commit.md

```diff
@@ -1,7 +1,6 @@
 # commit
 
-Record changes to DVC-tracked files in the <abbr>project</abbr>, by updating
-[DVC-files](/doc/user-guide/dvc-files-and-directories) and saving
+Record changes to DVC-tracked files in the <abbr>project</abbr>, by saving
 <abbr>outputs</abbr> to the <abbr>cache</abbr>.
```

Comment:
"updating DVC-files" was important. You just removed it, but it should be replaced with the new terminology.

Do you need help understanding the command? Please reach out to us directly in these cases to save time. I'm on Discord, for example.

Thanks